xbmc/interfaces/builtins/SystemBuiltins.cpp

```diff
@@ -148,6 +160,11 @@ static int Suspend(const std::vector<std::string>& params)
 ///     Starts the screensaver
 ///   }
 ///   \table_row2_l{
+///     <b>`InhibitScreensaver`</b>
+///     ,
+///     Inhibit the screensaver
+///   }
+///   \table_row2_l{
```

Comment:
Maybe make it a more detailed, e.g.:
```.cpp
///   \table_row2_l{
///     <b>`InhibitScreensaver(yesNo)`</b>
///     ,
///     Inhibit the screensaver
///     @param[in] yesNo   value with "true" or "false" to inhibit or allow screensaver (leaved empty force to true)
///   }
```

Reply:
Otherwise I find this very good and helpful for my web browser addon :blush: (to avoid screensaver on webvideo)

Reply:
@AlwinEsch : Thanks for the review I'll incorporate your suggestions in this PR.